test/test_torch.py

```diff
@@ -19094,6 +19095,22 @@ class TestTorchMathOps(TestCase):
 class TestTorch(AbstractTestCases._TestTorchMixin):
     exact_dtype = True
 
+# Tests that appropriate source range is highlighted in error messages.
+class TestHighlitedSource(unittest.TestCase):
+    def test_source_highlighted(self):
+        def to_list_missing_type_annotation(x):
+            # type: (torch.Tensor) -> List[float]
+            li = x.tolist()
+            return li
+
+        try:
+            m = torch.jit.script(to_list_missing_type_annotation)
+        except Exception as e:
+            FileCheck().check_source_highlighted("x.tolist").run(str(e))
```

Comment:
Rather than adding this here as a new test case, I was thinking that it could be part of the original `to_list_missing_type_annotation` test itself.

Reply:
Done.